docs/resources/dms_rocketmq_instance.md

```diff
@@ -0,0 +1,156 @@
+
+  Changing this parameter will create a new resource.
+
+* `flavor_id` - (Required, String, ForceNew) Specifies a product ID. The options are as follows:
+  + **c6.4u8g.cluster**: maximum number of topics on each broker: 4000; maximum number of consumer groups
+    on each broker: 4000
+  + **c6.8u16g.cluster**: maximum number of topics on each broker: 8000; maximum number of consumer groups
+    on each broker: 8000
+  + **c6.12u24g.cluster**: maximum number of topics on each broker: 12,000; maximum number of consumer groups
+    on each broker: 12,000
+  + **c6.16u32g.cluster**: maximum number of topics on each broker: 16,000; maximum number of consumer groups
+    on each broker: 16,000
+  Changing this parameter will create a new resource.
+
+* `storage_spec_code` - (Required, String, ForceNew) Specifies the storage I/O specification.
+  The options are as follows:
+  + **dms.physical.storage.high.v2**: high I/O disk
+  + **dms.physical.storage.ultra.v2**: ultra-high I/O disk
+  Changing this parameter will create a new resource.
+
+* `description` - (Optional, String) Specifies the description of the DMS RocketMQ instance.
+  The description can contain a maximum of 1024 characters.
+
+* `ssl_enable` - (Optional, Bool, ForceNew) Specifies whether the RocketMQ SASL_SSL is enabled. Defaults to false.
+  Changing this parameter will create a new resource.
+
+* `ipv6_enable` - (Optional, Bool, ForceNew) Specifies whether to support IPv6. Defaults to false.
+  Changing this parameter will create a new resource.
+
+* `enable_publicip` - (Optional, Bool, ForceNew) Specifies whether to enable public access.
+  By default, public access is disabled.
+  Changing this parameter will create a new resource.
+
+* `publicip_id` - (Optional, String, ForceNew) Specifies the ID of the EIP bound to the instance.
+  Use commas (,) to separate multiple EIP IDs.
+  This parameter is mandatory if public access is enabled (that is, enable_publicip is set to true).
+  Changing this parameter will create a new resource.
+
+* `broker_num` - (Optional, Int, ForceNew) Specifies the broker numbers. Defaults to 1.
+  Changing this parameter will create a new resource.
+
+* `retention_policy` - (Optional, Bool) Specifies the ACL access control.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - Specifies a resource ID in UUID format.
+
+* `status` - Indicates the status of the DMS RocketMQ instance.
+
+* `type` - Indicates the DMS RocketMQ instance type. Value: cluster.
+
+* `specification` - Indicates the instance specification. For a cluster DMS RocketMQ instance, VM specifications
+  and the number of nodes are returned.
+
+* `subnet_cidr` - Indicates the subnet route address.
+
+* `user_id` - Indicates the ID of the user who created the DMS RocketMQ instance.
+
+* `user_name` - Indicates the name of the user who created the DMS RocketMQ instance.
```

Comment:
I think the two attributes are meaningless

Reply:
done